Directory.Build.props

```diff
@@ -10,7 +10,7 @@
         <AnalysisLevel>latest</AnalysisLevel>
         <AnalysisMode>AllEnabledByDefault</AnalysisMode>
         <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-        <NoWarn>$(NoWarn);CA1014;CS8002;CA1416</NoWarn>
+        <NoWarn>$(NoWarn);CA1014;CS8002;CA1416;NU5104</NoWarn>
```

Comment:
This is a warning about the use of a preview package. This is the same Spectre package that CD is using in their latest version. It was needed to be able to access the ScanSettings Type.